kafka-commons/src/test/java/uk/gov/homeoffice/digital/sas/Constants/TestConstants.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.homeoffice.digital.sas.Constants;
+
+public class TestConstants {
+
+  public static final String KAFKA_VALID_VERSION = "0.1.0";
+
+  public static final String KAFKA_INVALID_VERSION = "1.0.0";
+
+  public static final String KAFKA_VALID_RESOURCE = "uk.gov.homeoffice.digital.sas.model.Profile";
```

Comment:
Isn't this more of a resource package/path rather than an actual resource? 
Possibly rename.